Foxtrot/Tests/TestDriver.cs

```diff
@@ -448,11 +457,11 @@ private static void CompareExpectedOutput(Options options, OutputCapture capture
                     actual = TrimFilePath(actual, options.SourceFile);
                     expectedLine = TrimFilePath(expectedLine, options.SourceFile);
                 }
-                Assert.AreEqual(expectedLine, actual);
+                Assert.Equal(expectedLine, actual);
             }
             if (expectedIndex < expected.Length)
             {
-                Assert.AreEqual(expected[expectedIndex], null);
+                Assert.Equal(expected[expectedIndex], null);
```

Comment:
`Assert.Null`? I saw you changed similar code before... So should we try to be consistent here as well? 

This stuff is minor, just IMO.